README.md

```diff
@@ -14,107 +14,87 @@ Assumptions:
 
-4. Switch to the alpha release version of the code. This is the latest stable release.
+3. Switch to the alpha release version of the code. This is the latest stable release.
 
         $ git checkout v0.1.0-alpha
 
     **Note:** If you'd rather use the latest, work-in-progress features of the DATA Act broker, replace the above command with `git checkout staging`.
 
-5. Change to the `data-act-broker-web-app` directory that was created when you cloned the DATA Act web repository.
+4. Change to the `data-act-broker-web-app` directory that was created when you cloned the DATA Act web repository.
 
-6. Install the web application's package dependencies:
+5. Install the web application's package dependencies:
 
         $ npm install
 
 
 ### Create Configurations
 
-The `data-act-broker-web-app` folder provides three sample `GlobalConstants` files:
-
- * `sampleGlobalConstants_dev.js`
- * `sampleGlobalConstants_local.js`
- * `sampleGlobalConstants_prod.js`.
+The `data-act-broker-web-app` folder provides a configuration object in the `GlobalConstants.js` file. By default, it is configured to point at `http://localhost:9999/v1/`. To modify your configuration, you may overwrite this file directly or use an environment specific file with the following naming convention instead: `GlobalConstants_<local/dev/prod>.js`.
 
-Use these sample files to create files named `GlobalConstants_dev.js`, `GlobalConstants_local.js`, and `GlobalConstants_prod.js` respectively.
-
-The sample files require you to provide values for:
+This configuration object contains the following properties:
 
 * `API` is the base API URL for the server that is hosting the API. It should start with an `https://` or `http://` protocol and end with `/v1/`, including the trailing slash
-
-	* Note: the `sampleGlobalConstants_local.js` already has this field configured for you.
-
 * `LOCAL_ROOT` is the URL from which you are serving the frontend (this can be left as an empty string for non-local usage).
 * `CAS_ROOT` is the root endpoint for the [CAS server](https://apereo.github.io/cas/4.2.x/protocol/CAS-Protocol-Specification.html#cas-uris) when a CAS single sign-on service is used.
 * `AUTH_CALLBACK` is the [callback URL](https://apereo.github.io/cas/4.2.x/protocol/CAS-Protocol-Specification.html#response) that the CAS server redirects to upon successful login.
 * `GA_TRACKING_ID` is the tracking ID for Google Analytics.
+* `LOCAL/DEV/PROD/STAGING` is the boolean value for which environment the `API` is targeting.
 
-Other fields, such as `LOCAL` and `DEV` should be left based on their sample values.
-
-### Run gulp tasks:
+### Run npm scripts:
 
-Several Gulp tasks are available to build the frontend web site for various use cases.
+Several npm scripts are available to build the frontend web site for various use cases.
 
 #### Hosted Production
 
 If you are building the web site for a hosted production environment, run:
 
 ```bash
-	$ gulp production
+	$ npm prod
```

Comment:
Should this be `npm run prod`? Same for other commands - they seem to only work with `run`

Reply:
👌